help/help.adoc

```diff
@@ -41,11 +68,11 @@ IMPORTANT: For all parameter descriptions below, required parameters are shown i
 Plugin configurations are sets of parameters that apply across some or all of the plugin's procedures. They are intended to reduce repetition of common values, create predefined sets of parameters for end users, and to securely store credentials where needed. Each configuration is given a unique name that is entered in designated parameters on procedures that use them.
 
 
-Plugin configurations are created by going to the {CD} "Administration" tab, then to the "Plugins" sub-tab. On the right side of the line for the EC-Azure plugin, there is a "Configure" link which will open the Configuration page.
+Plugin configurations are created by going to the {PRODUCT} "Administration" tab, then to the "Plugins" sub-tab. On the right side of the line for the EC-Azure plugin, there is a "Configure" link which will open the Configuration page.
 
 Subscription ID must be created and an X.509 management certificate needs to be uploaded to the certificate store. You can do this using the https://www.windowsazure.com[Management Portal]
 
-[cols=",",options="header",]
+[cols="1,1",options="header",]
```

Comment:
To ensure the tables are interpreted and rendered as AsciiDoc, recommend updating all tables to include the `a` and to remove the extra `,`: https://engineering.beescloud.com/docs/beewrite/latest/syntax/tables
```suggestion
[cols="1a,1a",options="header"]
```